### PR TITLE
[RORDEV-1193] taking into consideration OSS version of ES when the ROR settings are being configured

### PR DESCRIPTION
**Pull request by coutoPL**
🐞Fix (ES) no possibility to use ROR SSL with Elasticsearch OSS version

### CODE REVIEW

**Review by mateuszkp96 — APPROVED** [review]
LGTM

**Comment by mk-software-pl** [comment]
@mateuszkp96 it was a problem with non-docker-based installation, so I had to refactor the code a little bit. Could you please take a look?

**Comment by coutoPL** [comment]
@mateuszkp96 it was a problem with non-docker-based installation, so I had to refactor the code a little bit. Could you please take a look?

File: es717x/src/main/scala/tech/beshu/ror/es/ReadonlyRestEsConfig.scala [hunk 1]
```diff
@@ -21,20 +21,18 @@ import monix.eval.Task
 import tech.beshu.ror.configuration.{EnvironmentConfig, FipsConfiguration, MalformedSettings, RorBootConfiguration, RorSsl}
 import tech.beshu.ror.utils.ScalaOps._
 
-import java.nio.file.Path
-
 final case class ReadonlyRestEsConfig(bootConfig: RorBootConfiguration,
```
Comment (mateuszkp96) [inline es717x/src/main/scala/tech/beshu/ror/es/ReadonlyRestEsConfig.scala:24]: Let's move this class to the core module (I don't see any differences between current modules)

Comment (coutoPL) [reply es717x/src/main/scala/tech/beshu/ror/es/ReadonlyRestEsConfig.scala]: good catch 👍

**Review by mateuszkp96 — APPROVED** [review]
LGTM

**Pull request merged**